scripts/distribution/ubuntu-packages/template/debian/rules

```diff
@@ -2,17 +2,12 @@
 %:
 	dh $@
 
-# override_dh_update_autotools_config:
-# 		mkdir -p binaries
-# 		docker run -v $(shell pwd)/binaries:/out static-node-binaries cp /build/bin/concordium-node /out
-# 		docker run -v $(shell pwd)/binaries:/out static-node-binaries cp /build/bin/node-collector /out
-
 # install two services
 # They will be enabled and started automatically when installed.
 # Enabled means they will be started on boot.
 override_dh_installsystemd:
-		dh_installsystemd --name=concordium-node # --no-enable --no-start
-		dh_installsystemd --name=concordium-node-collector # --no-enable --no-start
+		dh_installsystemd --name=concordium-${build_env_name_lower}-node # --no-enable --no-start
```

Comment:
Any particular reasons that the options (which are commented out) are included?

Reply:
The reason is historical, but I left them in so that they are easy to discover. I'll make it into a proper comment instead.